specification/search/data-plane/Azure.Search/preview/2023-10-01-Preview/searchservice.json

```diff
@@ -6078,6 +6080,7 @@
           "x-nullable": false
         },
         "prioritizedFields": {
+          "x-ms-client-name": "semanticPrioritizedFields",
```

Comment:
This one could stay just `prioritizedFields`.  At least in .NET, we like making the type name super specific but don't mind if the property name is less verbose.

Reply:
I think this could be dropped back to `prioritizedFields`.